packages/core/app/styles/navi-core/components/navi-visualizations/table.scss

```diff
@@ -3,34 +3,34 @@
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 
-@navi-table-max-wrap: 3;
-@navi-header-line-height: 1.3em;
+$navi-table-max-wrap: 3;
+$navi-header-line-height: 1.3em;
+$font-family-strong: 'HelveticaNeueW01-55Roma', Helvetica, Arial, sans-serif;
```

Comment:
Should we remove this?

Reply:
Keeping for now because it looks better than without